web/modules/custom/dpl_webform/dpl_webform.install

```diff
@@ -55,3 +55,21 @@ function dpl_webform_update_10002(): string {
 
   return 'All translations for the webform ' . $webform_id . ' have been deleted.';
 }
+
+/**
+ * Delete translation of webform.settings.yml.
+ *
+ * Due to some faulty, prior translation, the danish version of this file
+ * has duplicate keys, which causes major issues on the site when loading
+ * the webform settings.
+ */
+function dpl_webform_update_10004(): string {
```

Comment:
```suggestion
function dpl_webform_update_10003(): string {
```

It seems like this jumps an update number?

Reply:
Not on purpose - fixed.